src/ptychi/reconstructors/pie.py

```diff
@@ -94,46 +92,53 @@ def compute_updates(
         obj_patches = self.forward_model.intermediate_variables["obj_patches"]
         psi = self.forward_model.intermediate_variables["psi"]
         psi_far = self.forward_model.intermediate_variables["psi_far"]
-        unique_probes = self.forward_model.intermediate_variables.shifted_unique_probes[0]
+        unique_probes = self.forward_model.intermediate_variables.shifted_unique_probes
 
         psi_prime = self.replace_propagated_exit_wave_magnitude(psi_far, y_true)
         # Do not swap magnitude for bad pixels.
         psi_prime = torch.where(
             valid_pixel_mask.repeat(psi_prime.shape[0], probe.n_modes, 1, 1), psi_prime, psi_far
         )
         psi_prime = self.forward_model.free_space_propagator.propagate_backward(psi_prime)
-
-        delta_o = None
-        if object_.optimization_enabled(self.current_epoch):
-            step_weight = self.calculate_object_step_weight(unique_probes)
-            delta_o_patches = step_weight * (psi_prime - psi)
-            delta_o_patches = delta_o_patches.sum(1, keepdim=True)
-            delta_o = ip.place_patches_integer(
-                torch.zeros_like(object_.get_slice(0)),
-                positions.round().int() + object_.pos_origin_coords,
-                delta_o_patches[:, 0],
-                op="add",
-            )
-            # Add slice dimension.
-            delta_o = delta_o.unsqueeze(0)
-
+        
+        delta_exwv_i = (psi_prime - psi)
+        delta_o = torch.zeros_like( object_.data )
```

Comment:
Please remove extra spaces on this line and the redundant parenthesis in the line above. Don't pad function arguments or array indices with spaces.

Reply:
done